src/richie/apps/courses/models.py

```diff
@@ -224,6 +228,61 @@ def copy_relations(self, oldinstance, language):
         self.courses.set(oldinstance.courses.drafts())
 
 
+class Licence(models.Model):
+    """
+    Licence model.
+
+    Instances of this models should only be created by administrators.
+    """
+
+    name = models.CharField(_("name"), max_length=200)
+    logo = FilerImageField(
+        related_name="licence",
+        verbose_name=_("logo"),
+    )
+    link_url = models.CharField(
+        _('link url'),
+        blank=True,
+        max_length=255,
+    )
+    description = models.TextField(
+        _(u"description"),
+        blank=True,
+        default="",
+    )
+
+    class Meta:
+        verbose_name = _("licence")
+
+    def __str__(self):
+        """Human representation of a person title"""
+        return "{model}: {name}".format(
+            model=self._meta.verbose_name.title(),
+            name=self.name,
+        )
+
+
+@python_2_unicode_compatible
```

Comment:
I think we do not expect this project to be python2 compatible.
There are other decorator alike in plugins

Reply:
Nice catch, self remember to remove this from other plugins i did